theme/templates/resource-landing-page/coverage.html

```diff
@@ -290,9 +302,17 @@ <h3>Coverage</h3>
                      </button>
                 {% endif %}
             </div>
-
-            <div class="col-xs-12"><h4>Temporal:</h4></div>
-
+            <div class="col-xs-12" style="margin-top: 20px;">
```

Comment:
can we move the inline style into stylesheet?

Reply:
will fix it.

Reply:
For spacing such as this, you can also use a tag for line breaks. That would be `<br>`